docs/09-Advanced.md

```diff
@@ -425,6 +445,8 @@ Plugins should derive from Chart.PluginBase and implement the following interfac
 	afterDatasetsUpdate: function(chartInstance) { }
 	afterUpdate: function(chartInstance) { },
 
+	configUpdate: function(chartInstance, newConfig) {},
```

Comment:
That was from an old revision. We decided not to have the `configUpdate` notification. Good catch though. I will update the docs